capa/rules/__init__.py

```diff
@@ -788,7 +856,10 @@ def from_dict(cls, d: Dict[str, Any], definition: str) -> "Rule":
         name = meta["name"]
         # if scope is not specified, default to function scope.
         # this is probably the mode that rule authors will start with.
+        # each rule has two scopes, a static-flavor scope, and a
+        # dynamic-flavor one. which one is used depends on the analysis type.
         scope = meta.get("scope", FUNCTION_SCOPE)
```

Comment:
```suggestion
        scopes = meta.get("scopes", FUNCTION_SCOPE)
```

as suggested above, maybe we make this plural.